rails-task-list/app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,55 @@
+<h1>Tasks for <%= @person.name %></h1>
+
+
+<a href="/tasks/new">
+<table id="add_task_button">
+  <tr>
+    <td>
+      <i class="fa fa-space-shuttle fa-4x"></i>
+    </td>
+    <td id="add_text">
+      Add a new task
+    </td>
+    <td>
+      <i class="fa fa-space-shuttle fa-4x fa-rotate-180 fa-flip-horizontal"></i>
+    </td>
+  </tr>
+</table>
+</a>
+
+<div id="button_box">
+
+  <div id="background_mod">
+
+
+  <% @tasks.each do |task| %>
+  <a id="task_link" href="/tasks/<%= task.id %>"
```

Comment:
If you are using this within an each loop, that means that this id will be set to the _same_ value for each task and we do not want to use the same ID more than once in our HTML.